pkg/solidity-utils/contracts/test/ERC4626TestToken.sol

```diff
@@ -39,6 +40,22 @@ contract ERC4626TestToken is ERC4626, IRateProvider {
         return _convertToAssets(FixedPoint.ONE, Math.Rounding.Floor);
     }
 
+    /**
+     * @notice Mints underlying and/or wrapped amount to the ERC4626 token.
+     * @dev If we set a rate to the ERC4626 token directly, we do not reproduce rounding issues when dividing assets
+     * by total supply. The best way to mock a rate is to inflate underlying and wrapped amounts.
```

Comment:
I'd just add a numerical example illustrating how to use it; seems like there's more than one way. To change it to 2, should you inflate one twice as much as the other, according to their existing balances? Or set an amount only on one, leaving the other 0?

Also consider a "helper" mockRate function that calls `inflateUnderlyingOrWrapped` in a standard way to set the rate, while retaining the ability to call the inflation directly for fine-tuned control.

Reply:
I'll add the mockRate suggestion, makes sense to use inflate only for fine-tuned control, as you mentioned